local_run_utils.py

```diff
@@ -30,8 +30,9 @@ def initialize_local_run(output_files_path, get_cfg: bool = True):
     if get_cfg:
         return create_validate_json_config_prov(output_files_path)
     
-def initialize_local_run_simulated(output_files_path, get_cfg: bool = True):
+def initialize_local_run_simulated(output_files_path, get_cfg: bool = True, launch_server=True):
     initialize_local_environment_simulated()
-    run_rasa_model_server(output_files_path)
+    if launch_server:
```

Comment:
I actually updated run_rasa_model_server to do this internally (not pushed into main yet but should be soon once reviewed!)
https://github.com/dialogue-planning/contingent-plan-executor/blob/74d99aebbc4a4a4ab7deb68e5361b138fbaacaac/contingent_plan_executor/local_run_utils.py#L16

Reply:
for similar reasons with the UI i had to make sure the server wasn't launched every time :P

Reply:
@beckydvn So if I make this change now, I'll be merging in a broken state of things into main which might not be the best method? Perhaps it would be better for when you merge your work to remove this parameter from this code in the same PR which implements it inside the run_rasa_model_server function? 

Let me know what you think.

Reply:
I think it would be easiest for you to just copy the version of the `run_rasa_model_server` from `modernize_hovor` into your code -- it was only a change of a couple lines and just changing that function will be enough :)